index.html

```diff
@@ -0,0 +1,99 @@
+    <link rel='stylesheet' href='normalize.css'>
+    <link rel='stylesheet' href='styles.css'>
+    <link href="https://fonts.googleapis.com/css?family=Oswald:200" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Raleway:200,300,400" rel="stylesheet">
+</head>
+<body>
+
+    <header>
+        <div class='container'>
+            <h1>JAMES KAPELLA</h1>
+            <h3>Full stack web developer</h3>
+            <img class='scrollingFingerImage pulsingImage' src='img/scrollFingerIcon.png' alt='Scroll-down icon'>
+            <section>
+                <h4>About me</h4>
+                <p>I’m a Full Stack Developer-in-training with The Mayden Academy who’s enthusiastic about both front-end design and back-end systems. I’m also a Certified Scrum Master with training in Agile methodologies. I’m fascinated by games and games design, and have been jamming and creating in my spare time for several years. Away from keyboards and screens I can also be found making loud noises with electric guitars, and stroking all the good dogs. </p>
+            </section>
+            <section>
+                <h4>Contact</h4>
+                <ul>
+                    <li>
+                        <a href='https://github.com/JKapella' target='_blank'>
+                            <img src='img/githubIcon.png' alt='a github icon'> Github
+                        </a>
+                    </li>
+                    <li>
+                        <a href='mailto:jvkapella@gmail.com' target='_blank'>
+                            <img src='img/emailIcon.png' alt='an email icon'> Email
+                        </a>
+                    </li>
+                    <li>
+                        <a href='http://www.linkedin.com/in/james-kapella-895001106' target='_blank'>
+                            <img src='img/linkedinIcon.png' alt='a linkedin icon'> LinkedIn
+                        </a>
+                    </li>
+                    <li>
+                        <a href='http://culturebosh.com' target='_blank'>
+                            <img src='img/gameIcon.png' alt='a games icon'> My Games
+                        </a>
+                    </li>
+                </ul>
+            </section>
+        </div>
+    </header>
+    <main>
+        <div class='container'>
+            <h2>PORTFOLIO</h2>
+            <h3>Scroll down</h3>
+            <img class='scrollingMouseImage pulsingImage' src='img/mousescrollitem.png' alt='Scroll-down icon'>
+            <a href='../w1_pilotShopRebuild/index.html' target='_blank'>
+                <article>
+                    <h3>PILOT SHOP</h3>
+                    <img src='img/pilotShopScreenshot.jpg' alt='A screenshot of my pilot shop rebuild'>
+                    <p>As an exercise in responsive web design we recreated the Pilot Shop digital storefront with HTML and CSS.</p>
+                </article>
+            </a>
+            <a href='' target='_blank'>
```

Comment:
best practice tip:
Fine this time so don't change, but hint for the future
a tags are inline elements and articles are block level elements
it is good practice not to put a block level element inside a default inline element. instead doing it the other way round, then changing the a tag to inline-block or block so you can assign a width and height

Reply:
Thanks, that makes a lot of sense, could probably have avoided using the article tags altogether and several lines of html.